src/index.js

```diff
@@ -265,7 +265,7 @@ Stream.prototype.join = function () {
 import { continueWith } from './combinator/continueWith'
 
 // @deprecated flatMapEnd, use continueWith instead
-export { continueWith, continueWith as flatMapEnd }
+export { continueWith, continueWith as alt, continueWith as flatMapEnd }
```

Comment:
We can't use `alt` as an alias for `continueWith`, since they have different types: `alt` accepts a Stream, and `continueWith` accepts a function that returns a Stream:

```hs
alt :: Stream a -> Stream a -> Stream a
continueWith :: Stream a -> (() -> Stream a) -> Stream a
```

However, `alt` _can_ be an alias for [`concat`](https://github.com/cujojs/most/blob/master/src/combinator/build.js#L23).  Note that `concat` is deprecated, which is ok.  We can remove the name `concat` in 2.0 and still keep `alt`.

Reply:
But `alt` then will be incorrect for streams like `concat` right now.

Reply:
Why should we keep it?

Reply:
Sorry, it was probably confusing that I said `alt` should be an alias for `concat`.  That implies that they have the same intention.  In my view, they don't have the same intention, even though their _implementations_ can be identical _temporarily_.

#### The problem with concat

The problem with concat, [as we've discussed](https://github.com/mostjs/core/pull/80), is the mental model of "concatenation", which tends to be associated with arrays and other index-based data structures.  That intuition doesn't work well for time-sensitive structures, where concatenation implies time-shifting.

#### The intention of alt

So, even if alt has the same _implemention_ as concat in 1.0, it has a different intention, that of "alternative", and we can (and should!) document it differently.  Rather than concatenation, the intention of alt is to observe one stream until it ends and then begin observing the other alternative stream.  Haskell's Netwire is my current mental model/precedent for alt.

#### Should we keep alt in most 1.x?

I don't mind including it in 1.x with carefully worded documentation.  If folks feel like it is confusing and not useful, the we probably shouldn't have alt in 1.x.

In most 2.0, which will be based on `@most/core`, I think alt will be more intuitive.  

#### Should we delete concat in 2.0?

From the perspective of a Semigroup, if we continue Fantasyland support in 2.0, IMHO we should change `concat` to be an alias for a binary `merge` operation (and keep `alt` as above)